dev-perl/html_parser/html_parser-3.71.recipe

```diff
@@ -8,27 +8,28 @@ COPYRIGHT="1995-2009 Gisle Aas
 	1999-2000 Michael A. Chase"
```

Comment:
I think this one does not need to be built for the secondary arch. It is a perl module and used only for the unit tests of NetSurf (not linked directly with it)